src/content/en/updates/2017/02/navigation-preload.md

```diff
@@ -0,0 +1,338 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Navigation preload lets you overcome service worker startup time by making requests in parallel.
```

Comment:
It has no impact on the startup delay, but it lets you do often-longer work in parallel, so the startup is no longer the bottleneck.

Let's say it takes me 10 minutes to walk to the restaurant, and it takes them 15 minutes to cook the meal. If I walk there, then order my food, it takes me 25 minutes to get food. If I walk slower, it takes me longer to get food. If I phone-ahead, then walk there, I get food in 15 minutes. It's unfair to say I've reduced the cooking time, nor am I reducing the time it takes me to get to the restaurant. If I walk slower, and it takes me 13 minutes to get to the restaurant, I still get food in 15 minutes.

So, I'm trying to say I've overcome the delay that walking to the restaurant introduces, as the bottleneck is now the food preparation time.